locales/pt.lua

```diff
@@ -36,39 +38,45 @@ local Translations = {
         ["added_garage"] = "Adicionaste uma garagem: %{value}",
         ["exit_camera"] = "Sair da câmera",
         ["house_for_sale"] = "Casa à venda",
-        ["decorate_interior"] = "Decorar Interior",
-        ["create_house"] = "Criar casa (somente imóveis)",
+        ["decorate_interior"] = "Decorar interior",
+        ["create_house"] = "Criar casa (somente agentes imobiliários)",
         ["price_of_house"] = "Preço da casa",
         ["tier_number"] = "Número do nível da casa",
-        ["add_garage"] = "Adicionar garagem residencial (somente imóveis)",
-        ["ring_doorbell"] = "Tocar a campainha"
+        ["add_garage"] = "Adicionar garagem residencial (somente agentes imobiliários)",
+        ["ring_doorbell"] = "Tocas a campainha"
     },
     menu = {
-        ["house_options"] = "Opções Da Casa",
-        ["enter_house"] = "Entrar Em Casa",
-        ["give_house_key"] = "Dara Chaves Da Casa",
-        ["exit_property"] = "Sair De Casa",
-        ["front_camera"] = "Câmera Da Frente",
+        ["house_options"] = "Opções da casa",
+        ["close_menu"] = "⬅ Fechar menu",
+        ["enter_house"] = "Entrar em casa",
+        ["give_house_key"] = "Dar chaves da casa",
+        ["exit_property"] = "Sair de casa",
+        ["front_camera"] = "Câmera da frente",
         ["back"] = "Voltar",
-        ["remove_key"] = "Remover Chave",
-        ["open_door"] = "Abrir Porta",
-        ["view_house"] = "Ver Casa",
-        ["ring_door"] = "Tocar À Campainha",
-        ["exit_door"] = "Sair De Casa",
-        ["open_stash"] = "Abrir Baú",
+        ["remove_key"] = "Remover chaves",
+        ["open_door"] = "Abrir porta",
+        ["view_house"] = "Ver casa",
+        ["ring_door"] = "Tocar à campainha",
+        ["exit_door"] = "Sair de casa",
+        ["open_stash"] = "Abrir baú",
         ["stash"] = "Baú",
-        ["change_outfit"] = "Alterar Outfit",
+        ["change_outfit"] = "Mudar outfit",
         ["outfits"] = "Outfits",
-        ["change_character"] = "Alterar Personagem",
+        ["change_character"] = "Alterar personagem",
         ["characters"] = "Personagens",
-        ["enter_unlocked_house"] = "Entrar Na Casa",
-        ["lock_door_police"] = "Trancar Porta"
+        ["enter_unlocked_house"] = "Entrar na casa,
```

Comment:
```suggestion
        ["enter_unlocked_house"] = "Entrar na casa",
```
Finish the string here

Reply:
ty